statsmodels/stats/multitest.py

```diff
@@ -444,3 +444,237 @@ def fdrcorrection_twostage(pvals, alpha=0.05, method='bky', iter=False,
         return reject, pvalscorr_, ntests - ri, alpha_stages
     else:
         return rej, pvalscorr, ntests - ri, alpha_stages
+
+
+def local_fdr(zscores, p0=1.0, null_density=None, deg=7,
+              nbins=30):
```

Comment:
It would be better to replace `p0` with `null_prop` or `null_proportion` (with prefix "null" similar to `null_density`)
An alternative would be to switch to post-fix `"_null"` in all names, but in this case I don't have a real preference.

we don't use "density" much in names, I guess `null_pdf` or `pdf_null` would be better, because we always refer to the function or method by name `pdf`